tests/bdd/steps/feature_steps/auth.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Authentication steps
```

Comment:
I can see that the value(s) for `Scenario`(s) within a feature-step file get used for the step names, but does the `Feature` value ever get used anywhere?

Reply:
The `Feature:` is required for the file to be grammatically correct, but the value isn't used anywhere by the system. It seems like a good place to put a description of what's in the file.

So I suppose it's used in the sense that a comment is used when you read it.

We could in future drop this comment in the generated file if we thought that was useful.